src/docs/release-notes/R22/R22.mdx

```diff
@@ -0,0 +1,54 @@
+### Brightlayer UI Migration
+
+The Power Xpert Blue "PX Blue" design system rebranded to Brightlayer User Interface "Brightlayer UI".
+
+-   [Brightlayer UI](https://brightlayer-ui.github.io/) documentation site
+-   Github organization migrated to [Brightlayer UI](https://github.com/brightlayer-ui)
+-   Component libraries migrated to [Brightlayer UI](https://brightlayer-ui-components.github.io/)
+-   NPM packages migrated to [Brightlayer UI](https://www.npmjs.com/org/brightlayerui)
+
+### Component Libraries
+
+-   Updated Angular Score Card component styles
+-   Updated React Info List Item with conditional title and subTitle rendering
+-   Added React Empty State card variation
```

Comment:
"Added React Empty State card variation"

Can you specify that this is for the storybook demo & not directly impacting the component library?

Reply:
yes, guess this too can be removed